### DISCUSSION TITLE
Qubino Shutter not updating the 23-38-0 Current position

### DISCUSSION THREAD

**Discussion by multinet33** [zwave-js/zwave-js-ui#3383] [Q&A]
Hello

I use ZWAVE JS UI on Home Assitant. I have 2 same QUBINO Shutter ZWNHCD (firmware 4.87).

I have 2 questions : 

1/ I don't understand why I get 2 different show on Meter v4 section 
--> Correct view on one qubino : 
![Capture d’écran 2023-10-27 à 21 28 04](https://github.com/zwave-js/zwave-js-ui/assets/17277695/ba251e4c-cf0e-46bf-ad55-ac0d103e29d5)
--> Wrong view on the other qubino : 
![Capture d’écran 2023-10-27 à 21 28 22](https://github.com/zwave-js/zwave-js-ui/assets/17277695/ca9e66a8-af14-429b-8b0f-f1555edcd61c)

2/ Second and PRINCIPAL problem 👍 
One qubino is not reporting the correct "23-38-0 currentValue" (the position) it always report the target position, but if I set target position to 0 and I stop the shutter at 60 the "23-38-0 currentValue" still have 0 (see screenshot below). 
On the other qubino the state is updated as expecting to 60

![Capture d’écran 2023-10-27 à 21 36 40](https://github.com/zwave-js/zwave-js-ui/assets/17277695/8993f31f-5abc-4237-b370-575ad7f3be57)

Can you help me ?

I have already done a reinterview, a clear cache and a refresh value with no success

Than you !

**Comment by multinet33** [comment]
Answer to myself

My principal problem (currentvalue position not updated) is fixed after a power off power on the qubino

If you have any information why the other problem exist (zwave js view does not show the same interface on 2 same qubino module

Thank you

**Reply by robertsLando** [reply]
@multinet33 A re-interview of the node can also help in such cases
